src/screens/Vault/AddSigningDevice.tsx

```diff
@@ -989,9 +978,33 @@ function Signers({
                   <>
                     {renderSigners(signerFilters)}
                     {signerFilters.length <= 0 && renderAssistedKeysShell()}
+                    <DashedCta
```

Comment:
We should leave the add key as it was at the top since it's not a primary action here, so leaving it as a small secondary action is better